### ISSUE TITLE
Add image alt texts

### ISSUE THREAD

**Issue by thatbudakguy** [Princeton-CDH/derrida-django#200]
- [x] Arrow corner logo (should be "home")
- [x] All cover images on browse/search pages (should be link destination)
- [x] All page images on browse/search pages (should be link destination)
- [x] "Has annotation" badge
- [x] "Not in collection" placeholder image
- [x] License image on library pages (get text from URI/json for the license)
- [x] Show/hide filters button

**Comment by bwhicks** [comment]
@rlskoeser For the cover/page images, the note says "should be link destination" as in the url or something more descriptive?

**Comment by rlskoeser** [comment]
@bwhicks documenting what we just discussed: should be a description of where the link will take you, e.g. "view bibliographic details" or "view TITLE details" or something like that.

**Comment by bwhicks** [comment]
This is available for someone who is not me to take a look at. There should be alt text on any image I could lay my hands on.

The one exception was the "Not available" image on the Citation list view. It uses background image trickery to produce the desired effect, so I made the `<span>` tab navigable with an `aria-label` to say what it means.

**Comment by rlskoeser** [comment]
Looks good to me. I'm going to tweak the wording in a few cases to make it a bit more consistent, and adding one more link title that I noticed, but calling this good.
